packages/storage-documents/src/on-demand-scan-request-message.ts

```diff
@@ -3,6 +3,6 @@
 
 export interface OnDemandScanRequestMessage {
     id: string;
+    batchRequestId: string;
```

Comment:
why are we adding batchRequestId?

Reply:
Originally we got batchRequestId from reading the scan run, which we aren't doing anymore

Reply:
we have deprecated batch calls. so we dont need this anymore.